seed/java-sdk/custom-auth/src/main/java/com/seed/customAuth/resources/errors/errors/SeedCustomAuthUnauthorizedRequest.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * This file was auto-generated by Fern from our API Definition.
+ */
+package com.seed.customAuth.resources.errors.errors;
+
+import com.seed.customAuth.core.SeedCustomAuthApiError;
+import com.seed.customAuth.resources.errors.types.UnauthorizedRequestErrorBody;
+
+public final class SeedCustomAuthUnauthorizedRequest extends SeedCustomAuthApiError {
+    private final UnauthorizedRequestErrorBody body;
+
+    public SeedCustomAuthUnauthorizedRequest(UnauthorizedRequestErrorBody body) {
+        super("UnauthorizedRequest", 401, body);
+        this.body = body;
+    }
+
+    @Override
+    public UnauthorizedRequestErrorBody body() {
+        return this.body;
+    }
```

Comment:
@dcb6 what happens when you `toString` one of these errors?

Reply:
or `System.out.printlin(e)`

Reply:
Same thing right? 
```java
    @Test
    public void test() {
        // Add tests here and mark this file in .fernignore
        UnauthorizedRequestErrorBody body = UnauthorizedRequestErrorBody.builder().message("Oh no!").build();
        SeedCustomAuthUnauthorizedRequest error = new SeedCustomAuthUnauthorizedRequest(body);
        System.out.println(error);
        assert true;
    }
```
prints:
```
SeedCustomAuthApiError{statusCode: 401, body: {
  "message" : "Oh no!"
}}
```

Reply:
was wondering if we should just do a `writeValueAsString` with an object mapper instead?

Reply:
I think the best way to improve the `toString` is just to add a message field like so:
```java
public class SeedCustomAuthApiError extends SeedCustomAuthError {

    private final String message;
    private final int statusCode;

    private final Object body;

    public SeedCustomAuthApiError(String message, int statusCode, Object body) {
        super(message);
        this.message = message;
        this.statusCode = statusCode;
        this.body = body;
    }

    public int statusCode() {
        return this.statusCode;
    }

    public Object body() {
        return this.body;
    }

    @java.lang.Override
    public String toString() {
        return "SeedCustomAuthApiError{" + "statusCode: " + statusCode + ", body: " + body +", message: " + message + "}";
    }
}
```

Not exactly sure how i'd cleanly refactor the `createToStringMethod` here to instead call getMessage() if we wanted to go that route: https://github.com/fern-api/fern/blob/dc03bc01865cfdeb9640b4b88d5dae106f0c36da/generators/java/generator-utils/src/main/java/com/fern/java/ObjectMethodFactory.java#L87

Reply:
`SeedCustomAuthError` as is prints`com.seed.customAuth.core.SeedCustomAuthError: Oh no!` for example (roughtly, `<className> <message>`)

so i don't think any improvements are needed there (i believe you mentioned adding a `message` field)